src/lib/utils/check_table_equal.hpp

```diff
@@ -11,8 +11,8 @@ namespace opossum {
 enum class OrderSensitivity { Yes, No };
 
 /**
- * "Strict" enforces that both tables have precisely the same column types, "Lenient" allows float instead of double, double
- * instead of float, long instead of int, int instead of long
+ * "Strict" enforces that both tables have precisely the same column types, "Lenient" allows float instead of double,
+ * double instead of float, long instead of int, int instead of long
```

Comment:
Generally, I'd like for PR comments to only refer to things actually changed by that PR, not formatting changes etc. Otherwise people might get discouraged to work on existing unclean code because they fear they might be made responsible for the entire mess in their PR.

Anyway, added "We need this for comparing Hyrise with SQLite 
 * since the column types of the latter might differ from Hyrise's."

Reply:
I try not to place an undue burden upon whoever opens the PR when it comes to improving legacy code. At the same time, opening an issue for something this small seems like overkill, too.

I’ll keep your concern in mind and will both add “(or open an issue)” to more of the comments and try to limit the requests that actually take time.

Reply:
I'd just ignore minor things.